data/strategies/StrategyAH2EarncyUSDC.json

```diff
@@ -0,0 +1,7 @@
+{
+  "$schema": "strategy",
+  "name": "StrategyAH2EarncyUSDC",
+  "description": "Lends USDC on Alpha Homora v2 to generate interest. Users of Alpha Homora borrow USDC to perform leveraged yield-farming on Alpha Homora’s platform.",
```

Comment:
I haven't seen any reference to outside users in other descriptions. It might confuse some people

Reply:
Good point